app/src/main/java/com/solux/flory/presentation/profile/ProfileFragment.kt

```diff
@@ -33,8 +33,8 @@ class ProfileFragment : BindingFragment<FragmentProfileBinding>(FragmentProfileB
 
     private fun neighborsTextClick() {
         binding.tvProfileNeighbors.setOnClickListener {
-            //이웃 목록 페이지로 이동
-
+            val intent = Intent(requireContext(), NeighborsActivity::class.java)
+            startActivity(intent)
```

Comment:
```suggestion
            startActivity( Intent(requireContext(), NeighborsActivity::class.java))
```
이런 식으로 코드 짤 수 있습니다!! 불필요한 변수 생성 지양하기!! 저두 나중에 이렇게 리팩할 예정입니다